### PR TITLE
Added basics CONTRIBUTING.md

### LINKED ISSUE #226: Missing CONTRIBUTING.md
Generally working groups have a CONTRIBUTING.md file. I was going to point an individual to it who asked about joining the WG's discussions, but it seems there isn't one at present 😅

### PR DESCRIPTION
**Pull request by mcollina**
Fixes #226 

This adds the basics.

cc @bnb @watson

### CODE REVIEW

**Comment by mcollina** [comment]
@richardlau good catch, I've removed them.

**Comment by watson** [comment]
Regarding the code of conduct part, I recommend also having a `CODE_OF_CONDUCT.md` as [GitHub will detect that](https://help.github.com/articles/adding-a-code-of-conduct-to-your-project/) and add links to it on relevant pages

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+The [Node.js Moderation Policy] applies to this WG.
+
+# Code of Conduct
+
+The [Node.js Code of Conduct][] applies to this WG.
+
+[Node.js Code of Conduct]:
+https://github.com/nodejs/node/blob/master/CODE_OF_CONDUCT.md
+[Node.js Moderation Policy]:
+https://github.com/nodejs/TSC/blob/master/Moderation-Policy.md
```
Comment (mhdawson) [inline CONTRIBUTING.md]: It think this should refer to the copy in the admin repo

**Comment by bnb** [comment]
LGTM 😁

**Pull request merged**